### ISSUE TITLE
Disable DNS,DHCP,TFTP installer config in 6.1 

### ISSUE THREAD

**Issue by pgagne** [RedHatSatellite/satellite-clone#74]
Currently we don't disable dhcp,dns,tftp in 6.1. We should add the following to the installer command line:

```
--capsule-dns false --capsule-dhcp false --capsule-tftp false
```

Having these enabled is dangerous and may cause issue. The user is expected to manually configure these after the clone is complete.

**Comment by sthirugn** [comment]
Also README should be updated to reflect this info

**Comment by pgagne** [comment]
This will probably also fix #13 and make it so we don't have to merge #63
